charts/grafana/templates/grafana-deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             - name: GF_DATABASE_URL
               valueFrom:
                 secretKeyRef:
-                  name: {{ template "grafana.backend.secret" . }}
+                  name: astronomer-bootstrap
```

Comment:
use this instead of {{ template "grafana.backend.secret" . }} because {{ template "grafana.backend.secret" . }} is not initialized until the next init container is done